README.md

```diff
@@ -25,3 +25,5 @@ Git command explanation:
 `git commit readme.md` Will commit the README file to the repository!
 
 'git branch <branch name>' (this will create a new branch with the desiganted name)
+
+'git switch <branch name> This command switches between different branches in git
```

Comment:
Please use backticks to wrap the command. 
<img width="622" alt="Screenshot 2023-02-25 at 10 19 03 AM" src="https://user-images.githubusercontent.com/2203934/221365021-12906508-1b33-4b7e-9caf-b153f8ae67d5.png">
Should be:
```
`git switch <branch name>` This command switches between different branches in git.
```